pkg/kubelet/stats/pidlimit/pidlimit_linux.go

```diff
@@ -25,19 +25,25 @@ import (
 	"syscall"
 	"time"
 
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	statsapi "k8s.io/kubelet/pkg/apis/stats/v1alpha1"
 )
 
 // Stats provides basic information about max and current process count
 func Stats() (*statsapi.RlimitStats, error) {
 	rlimit := &statsapi.RlimitStats{}
 
-	if content, err := ioutil.ReadFile("/proc/sys/kernel/pid_max"); err == nil {
-		if maxPid, err := strconv.ParseInt(string(content[:len(content)-1]), 10, 64); err == nil {
-			rlimit.MaxPID = &maxPid
+	var taskMax int64
+	for _, file := range []string{"/proc/sys/kernel/pid_max", "/proc/sys/kernel/threads-max"} {
```

Comment:
I recommend adding a comment to explain what is being calculated (a minimum for two sysctls, `kernel.pid_max` and `kernel.threads-max`), and _why_.